tailcfg/tailcfg.go

```diff
@@ -74,7 +74,8 @@ type CapabilityVersion int
 //	37: 2022-08-09: added Debug.{SetForceBackgroundSTUN,SetRandomizeClientPort}; Debug are sticky
 //	38: 2022-08-11: added PingRequest.URLIsNoise
 //	39: 2022-08-15: clients can talk Noise over arbitrary HTTPS port
-const CurrentCapabilityVersion CapabilityVersion = 39
+//  40: 2022-08-15: added CapabilityDataPlaneAuditLogs
```

Comment:
formatting

Reply:
CapVer should only be incremented when the Capability is used. Fine to add Capability without bumping the CapVer.

Reply:
sorry for doubling the comment